extensions/scarb-snforge-test-collector/tests/test.rs

```diff
@@ -1,6 +1,7 @@
 use assert_fs::prelude::*;
 use assert_fs::TempDir;
 use cairo_lang_sierra::program::StatementIdx;
+use cairo_lang_sierra_generator::statements_code_locations::SourceCodeSpan;
```

Comment:
Then why import it here?

Reply:
Because I embrace inconsist opinions (Good catch, I will change it)

Reply:
You can leave it as is :)